internal/driver/glfw/window.go

```diff
@@ -273,7 +274,7 @@ func (w *window) fitContent() {
 		return
 	}
 
-	if w.viewport == nil {
+	if w.viewport == nil || w.closing {
```

Comment:
consider adding `w.isClosing()` which could encapsulate these two tests and be used in several call sites in this file